db/schema.rb

```diff
@@ -802,6 +801,7 @@
     t.integer "unspent_credits_count", default: 0, null: false
     t.datetime "updated_at", null: false
     t.string "url"
+    t.datetime "latest_article_updated_at"
```

Comment:
As mentioned in Slack, you can exclude this file from the PR.

Reply:
reverted this a few commits ago - must have crossed with your review :)